app/account/page.tsx

```diff
@@ -0,0 +1,29 @@
+'use client';
+
+import Link from 'next/link';
+import GetInfo from './DisplayInfo';
+
+const AccountForm = () => (
+  <div className='form-widget'>
```

Comment:
what is this `form-widget` class? I don't see it in the tailwind docs and it doesn't appear to have been defined in our css

Reply:
That was just something in the first iteration of the code that I was messing around with, it is not too relevant and i can change it to <form>